src/components/Comments/WriteComment.js

```diff
@@ -190,7 +189,7 @@ export const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'column',
     // alignItems: 'center',
-    backgroundColor: colors.lightgrey,
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of a style object has been updated. Ensure that this new color is consistent with your design system.
```diff
- backgroundColor: colors.lightgrey,
+ backgroundColor: COLORS.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->